tests/src/test/java/org/eclipse/hono/tests/http/HttpTestBase.java

```diff
@@ -896,6 +898,109 @@ public void testUploadMessagesWithTtdThatReplyWithCommand(final TestContext ctx)
                 });
     }
 
+
+    /**
+     * Verifies that the queueDrainHandler is invoked on available of credit and a command is sent.
+     *
+     * @param ctx The test context.
+     * @throws InterruptedException if the test fails.
+     */
+    @Test
+    public void testQueueDrainHandler(final TestContext ctx) throws InterruptedException {
```

Comment:
this test looks very big in comparison to the very little functionality that we actually want to verify. Given that all of the relevant functionality has already been verified in the unit tests, I believe that we can safely remove this test.

Reply:
Done.